### ISSUE TITLE
Missing status tooltips in open CR table

### ISSUE THREAD

**Issue by amritkv** [eclipse-sw360/sw360-frontend#304]
Missing tooltips in open CR table:

	Status Tooltip :
		NEW :	
			CR is pending to be Accepted by the Clearing Team.
			A sanity check for the artifacts required for Clearing will be done prior to Accepting a CR.
		Sanity Check :
			The Clearing team is conducting a sanity check on the inputs provided by the project team.
			At this stage, they verify the availability of the source code for all linked releases.
		Accepted :
			CR has passed SanityCheck and is accepted into the clearing queue.
		Closed :
			Clearings requested in the CR is complete. (Closing resets the CR priority)
		Rejected :
			CR has failed SanityCheck. Contact Clearing Team. (Rejecting resets the CR priority)
		In Queue :
			CR is next in queue for clearing.
		In Progress :
			CR is being worked on by the clearing team.
		Awaiting Response :
			Clearing Team is waiting for input/action from the Requesting User.
		On Hold :
			Clearing Request is on hold
		Pending Input :
			The input needed for clearing is incomplete.
			The clearing team cannot proceed to the next stage of processing the CR until the project provides the missing input.

**Comment by amritkv** [comment]
PR merged in community ! https://github.com/eclipse-sw360/sw360-frontend/pull/308
